inc/compatibility/namespace.php

```diff
@@ -0,0 +1,46 @@
+<?php
+/**
+ * Compatibility tweaks for optimal performance.
+ *
+ * @package altis/core
+ */
+
+namespace Altis\Compatibility;
+
+/**
+ * Bootstrap compatibility tweaks.
+ */
+function bootstrap() {
+	add_action( 'plugins_loaded', __NAMESPACE__ . '\\set_woocommerce_compatibility' );
+}
+
+/**
+ * Set hooks for WooCommerce compatibility and optimization.
+ *
+ * Adjusts Action Scheduler to run optimially with Cavalcade, by avoiding
+ * fallbacks and maximising the Cavalcade queue.
+ *
+ * The time limit and batch size hooks are set at priority 0 to allow sites to
+ * easily override for their specific use case.
```

Comment:
```suggestion
 * easily override for their specific use case.
 *
 * Specifically we disable the action_scheduler_allow_async_request_runner. This is functionality in
 * Action Scheduler that will fall back to making async requests to admin-ajax.php to process the
 * queued jobs. This will typically get triggered if the Cron is expected not to be working. The issue we've
 * seen with this approach is when Cavalcade has a somewhat long queue, Action Scheduler switches to the 
 * fallback behavior, which we don't want. We don't want that because it eats up available PHP workers, where
 * we prefer to just wait for Cavalcade to get to the queued jobs.
 *
 * We're also adjusted the limits on how many Action Scheduler Jobs will run in a single Action Schedule
 * Cron Job as it's more efficient to run that way when we know how long Cavalcade jobs can run as opposed
 * to the default WordPress Cron runner which likely has lower timeouts.
```

Reply:
This is preeeeettttyyyy wordy, and kinda repeats what the code says; how about:

```suggestion
 * easily override for their specific use case.
 *
 * We disable the async task runner entirely, to avoid the loopback HTTP
 * requests to admin-ajax.php. This triggers when AS detects cron not working,
 * but doesn't account for the Cavalcade backlog and scaling system, and
 * triggers too eagerly. Instead, we prefer a slight delay as Cavalcade scales
 * up, avoiding the loopbacks.
 *
 * The limits are adjusted to account for the better processing power and
 * limits of Cavalcade.
```